src/dcurl.c

```diff
@@ -48,6 +51,11 @@ extern ImplContext PoWCL_Context;
 extern ImplContext PoWFPGAAccel_Context;
 #endif
 
+#if defined(ENABLE_REMOTE)
+extern RemoteImplContext Remote_ImplContext;
```

Comment:
Change the variable name `Remote_ImplContext` to `Remote_Context` for consistency.

Reply:
How about renaming `RabbitMQ_Context`? This is because `Remote_ImplContext` can be implemented by using ZeroMq or others.

Reply:
Do we need to execute multiple message queue implementation at the same time?
If not, I guess using `Remote_Context` is fine.
Since compare to the file `src/pow_c.c`, the variable `PoWC_Context` indicates the PoW implementation rather than the message queue implementation.
It might be a little confusing if we use `RabbitMQ_Context`.

Reply:
How about renaming `RemoteRabbitmq_Context`?